### ISSUE TITLE
Remove the describedBy regex enforcement

### ISSUE THREAD

**Issue by rolando-ebi** [HumanCellAtlas/metadata-schema#403]
**For which schema is a change/update being suggested?**

For all schemas, remove the regex restriction on `describedBy`

**What new field(s) need to be changed/added?**

none

**Why is the change requested?**

1. The regex on `describedBy` is pointless from the point of view of enforcing that these schemas came from schema.humancellatlas.org. A user could just take the existing schemas, host the schemas elsewhere and modify the regex to reflect the new URI, or delete the regex entirely.

2. The regex doesn't enforce the module/type/etc. of the schema in question, because the regex can be averted in a similar fashion as in 1)

3. The regex severely limits the reusability of schemas. If I want to re-use these schemas and host them on my own schema server, I'd be forced to change or delete the `describedBy` regex in every schema. For example, if the NeMO project wanted to re-use and modify these schemas in their own DCP, with their own schema server.

4. The regex severely limits the ability of ingest devs to test schemas that aren't yet published to schema.humancellatlas.org

**Comment by malloryfreeberg** [comment]
@rolando-ebi do you consider this something we need for the phase 1 MVP? If so, we'll prioritize it. If not, we'll put it on hold until after phase 1.

**Comment by lauraclarke** [comment]
@malloryfreeberg @rolando-ebi Is this needed or can I put this on ice?

**Comment by zperova** [comment]
@rolando-ebi is this still relevant and if so how important and how urgent is this?

**Comment by rolando-ebi** [comment]
@zperova this issue is still relevant in that it's not fixed. It's currently not urgent and isn't blocking anything.

**Comment by zperova** [comment]
thanks for an update @rolando-ebi

**Comment by ESapenaVentura** [comment]
This clashes with discussions about wrong environments being pointed out.

We can close this as this is not the direction we're taking.
